app/app.rb

```diff
@@ -11,10 +11,10 @@
 get '/' do
   appearance = {
     apiversion: "1",        
-    author: "",           # TODO: Your Battlesnake Username
-    color: "#888888",     # TODO: Personalize
-    head: "default",      # TODO: Personalize
-    tail: "default",      # TODO: Personalize
+    author: "JZ",           # TODO: Your Battlesnake Username
```

Comment:
Please also remove your customizations from this PR :)

Reply:
Whoops. Yes I'll get on this!